### ISSUE TITLE
Obtaining the current video playback quality

### ISSUE THREAD

**Issue by caida-ricky** [vimeo/player.js#496]
### Expected Behavior
- A method can provide the current video playback quality (4K, 2K, 1080p...).

### Actual Behavior
- No such a method

**Comment by fisherinnovation** [comment]
Thanks for your feedback! 

While we review this request internally I wanted to also provide a solution that may unblock you for the time being. The `getVideoWidth` and `getVideoHeight` methods can be used to get the source content resolution which you could utilize to calculate what quality string you would append to that video. 

See,
- https://github.com/vimeo/player.js#getvideowidth-promisenumber-error
- https://github.com/vimeo/player.js#getvideoheight-promisenumber-error

**Comment by luwes** [comment]
Duplicate of #601
